mvpa2/generators/permutation.py

```diff
@@ -94,6 +97,10 @@ def _get_pcfg(self, ds):
         return get_limit_filter(self._limit, collection)
 
 
+    def get_chunk_attr(self, ds):
```

Comment:
not sure what is the workflow between you and @hanke but now he got a PR https://github.com/hanke/PyMVPA/pull/8 ;-)